src/components/anchor-navigation/anchor-navigation.spec.tsx

```diff
@@ -243,23 +262,28 @@ describe("AnchorNavigation", () => {
   ])(
     "scroll triggers selection of proper navigation item based on the navigation and sections top position",
     (scrollPosition, selectedAnchorIndex) => {
+      const SECTION_VISIBILITY_OFFSET = 200;
```

Comment:
If we move these to the top of the file it will avoid initialising them on each loop